crypto/acrastruct.go

```diff
@@ -50,6 +52,7 @@ func (handler AcraStructHandler) Decrypt(data []byte, context *base.DataProcesso
 	privateKeys, err := context.Keystore.GetServerDecryptionPrivateKeys(accessContext.GetClientID())
 	defer utils.ZeroizePrivateKeys(privateKeys)
 	if err != nil {
+		base.AcrastructDecryptionCounter.WithLabelValues(base.LabelStatusFail).Inc()
```

Comment:
hm, as I understand, it will increment twice for the acra-translator for its `decrypt` methods? If AT uses cryptohandler that uses this handler under the hood, then it will increment twice